netlify/edge-functions/serve-definitions.ts

```diff
@@ -0,0 +1,20 @@
+import type { Context } from "netlify:edge";
+
+export default async (request: Request, context: Context) => {
+
+  // Deleting Origin header, which is involved in the cache policy, so requests can hit GH cache.
+  // Reason: raw.githubusercontent.com responses include vary: Authorization,Accept-Encoding,Origin
+  request.headers.delete("origin");
+
+  // Setting GH Token to increase GH rate limit to 5,000 req/h.
+  request.headers.set("Authorization", "token " + Deno.env.get("GITHUB_TOKEN"));
```

Comment:
We would need to use a GH token here. Preferably, a token from a user is not being used in any automation yet.  cc @derberg

Reply:
> a token from a user

what do you mean? why user token. We can use AsyncAPI token

Reply:
Oh, i meant using a new user because requests to GH API are counted across the whole account, no matters wich toke you use. Important for the rate limit.